app/components/publications/publication/case/info/publication-case-info-panel.hbs

```diff
@@ -162,7 +162,7 @@
                   {{else}}
                     <Auk::Datepicker
                       @date={{@publicationFlow.openingDate}}
-                      @onChange={{this.setOpeningDate}}
+                      @onChange={{set this "args.publicationFlow.openingDate"}}
```

Comment:
`this.args` is `@` in handlebars. Dus `{{set @publicationFlow "openingDate"}}` is hier logischer.

Reply:
Aha, ik dacht dat die `set` altijd binnen die `this` moest blijven. Goed dat je dit opmerkt!